edit-hopper.php

```diff
@@ -35,9 +35,19 @@ public function __construct() {
 		}
 
 		function init($screen) {
+			$box_title = "Edit something";
+                        switch ($screen) {
+                            case "post":
+                                $box_title = "Edit Posts";
+                                break;
+                            case "page":
+                                $box_title = "Edit Pages";
+                            default:
+                                break;
+                        }
 			add_meta_box(
 		        'edit_hopper_box',
-		        __( 'Edit Pages', 'edit_hopper_textdomain' ),
+		        __( $box_title, 'edit_hopper_textdomain' ),
```

Comment:
Right. It should work just fine for pages and posts. The only thing I can see being a problem, is later on down the road when we add custom post types and options to show it on those as well. Those could be created with dashes in the name which would show up in the meta box title. We could probably query WP, and find the title of the custom post type and drop that in instead.